src/components/renderField.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function renderField(props) {
```

Comment:
Let’s name these components the same as we would name any other, e.g. this one should probably be called something like `<AssignmentCreatorTextField>` and the next one `<AssignmentCreatorSelectField>` etc.